pharmd-app/src/components/Fields/TemporaryField.js

```diff
@@ -1,21 +1,19 @@
-//  This field is strictly for demonstration purposes only
-
 import React from "react";
 import tw, { css, styled } from "twin.macro";
 
 const Field = styled.span`
   ${tw`fontStyle-6 text-gray-700 font-medium`}
-  
-  ${props => props.primary && css`
-    ${tw`fontStyle-6 text-secondary font-semibold`}
-  `}
+
+  ${props =>
+    props.primary &&
+    css`
+      ${tw`fontStyle-6 text-secondary font-semibold`}
+    `}
   font-size: 1.4em;
 `;
 
 const TempField = ({ source, isEmphasis }) => {
-    return (
-      <Field primary={isEmphasis}>{source}</Field>
-    );
+  return <Field primary={isEmphasis}>{source}</Field>;
```

Comment:
Similar to TempGradeField, I was wondering if TempField should be:
```const TempField = ({ source, isEmphasis }) => <Field primary={isEmphasis}>{source}</Field>;```

Reply:
i think this is also a eslint thing